### DISCUSSION TITLE
Should I report failures with Wayland?

### DISCUSSION THREAD

**Discussion by peterscottie** [input-leap/input-leap#1929] [Q&A] [answered]
Ive down loaded the lastest version from git , and unsurprisingly it does really do much . In preparation for filling a bug report I enabled Debug1 , In the log I saw a message that seems to imply that you know it doesnt work with wayland. so my question is am i wasting your time reporting a bug?

Many thanks

**Comment by shymega** [comment, marked as answer]
It works with Wayland on KDE and GNOME. We want to know about those, provided the issue opened isn't a duplicate. Check the tracker for dupes.

On `wlroots`-based compositors, we are unlikely to ever support those.

**Comment by peterscottie** [comment]
On Sat, 29 Jun 2024 at 23:27, Dom Rodriguez ***@***.***>
wrote:

> It works with Wayland on KDE and GNOME. We want to know about those,
> provided the issue opened isn't a duplicate. Check the tracker for dupes.
>
> On wlroots-based compositors, we are unlikely to ever support those.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/input-leap/input-leap/discussions/1929#discussioncomment-9913921>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQD743QQGUTXFHCOB3IYJLTZJ4RENAVCNFSM6AAAAABKDOXS7SVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSMJTHEZDC>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

```
section: screens
	E-CND1487W6S:
		meta = shift
		halfDuplexCapsLock = false
		halfDuplexNumLock = false
		halfDuplexScrollLock = false
		xtestIsXineramaUnaware = false
		preserveFocus = false
#		switchCorners = True
		switchCornerSize = 50
	fedora:
		meta = shift
		halfDuplexCapsLock = false
		halfDuplexNumLock = false
		halfDuplexScrollLock = false
		xtestIsXineramaUnaware = false
		preserveFocus = false
#		switchCorners = true 
		switchCornerSize = 50
end

#sectionnput config : aliases
#.	E-CND1487W6S:
#		work
#end

section: links
	E-CND1487W6S:
		right = fedora
	fedora:
		left = E-CND1487W6S
end

section: options
	relativeMouseMoves = false
	screenSaverSync = true
	win32KeepForeground = false
	clipboardSharing = true
#	switchCorners = true 
	switchCornerSize = 50
end
```

**Comment by shymega** [comment]
I'm not sure why you've posted this here. I tried to edit it so it would render in code blocks - seems to be a GitHub limitation via email.

Please provide further insight into your issue, so that I can help.

**Comment by shymega** [comment]
Oh, you have opened a bug report. But you posted the config here. Please attach the config in a code block on your issue, then we can help there. I am going to close this discussion.
